docs/application/native/guides/device/battery-monitor.md

```diff
@@ -79,90 +77,105 @@ To get the battery usage information related to the application ID:
     }
     ```
 
-- Get total battery usage of an application ID over a specific duration of time.
-  To get a particular time duration type, use the `battery_monitor_duration_type_e` enum:
+- Get the total battery usage of an application ID over a specific interval of time. The timestamps **from** and **to** must be set as per the Unix Epoch time format:
 
     ```
-    duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
-    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    /* To get the usage for past 24 hours from the current epoch time */
+    int period_t = 24*60*60; //24 hours in seconds
+    to = now; from = to - period_t;
+
+    error_code = battery_monitor_get_power_usage_by_app(app_id, from, to, &battery_usage);
     if (error_code == BATTERY_MONITOR_ERROR_NONE)
-        printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+        printf("The total battery usage for appid [%s] over last 24 hrs is [%lf]", app_id, battery_usage);
```

Comment:
In other cases, the printed log is "The Battery Usage for appid....".

However in this case the printed log is "The total battery usage for appid...."

How about unifying the usage of "Battery Usage"? For example as like "The Total Battery Usage for appid...."

Same issue to line 101.